src/ai/susi/server/api/aaa/SignUpService.java

```diff
@@ -222,6 +225,14 @@ public ServiceResponse serviceImpl(Query post, HttpServletResponse response, Aut
 				result.put("message", "You successfully signed-up, but no email was sent: " + e.getMessage());
 			}
 		} else {
+			// store the time of signup in accounting object
+			Accounting accounting = DAO.getAccounting(identity);
+			Date currentTime = new Date();
+			SimpleDateFormat signupTime = new SimpleDateFormat("EEE, MMM d, yyyy hh:mm:ss a z");
```

Comment:
Please do not define the date format here. Use ai.susi.tools.DateParser.FORMAT_RFC1123 instead